erpnext/stock/dashboard/item_dashboard.js

```diff
@@ -82,6 +82,15 @@ erpnext.stock.ItemDashboard = Class.extend({
 			var message = __(" Currently no stock available in any warehouse")
 			$("<span class='text-muted small'>"+message+"</span>").appendTo(this.result);
 		}
+
+		if (roles.indexOf("Stock Manager") > -1 || roles.indexOf("Stock User") > -1){
+			$(".btn-move").show();
+ 			$(".btn-add").show();
+		} else {
```

Comment:
Dear: Vinayak
I think this is the right place for my question, I want to hide the (BTN-add) from 'sales user' and 'stock user'
please kindly help out how I can do that.
I've tried the perm level, set perm level to 'stock Manager' at 2, 'stock user' at 1, but still the the bth 'add' shows to everyone who has access to ITME.